storage/wdpost_run.go

```diff
@@ -467,7 +467,7 @@ func (s *WindowPoStScheduler) runPost(ctx context.Context, di dline.Info, ts *ty
 
 				skipCount += sc
 
-				ssi, err := s.sectorsForProof(ctx, good, partition.Sectors, ts)
+				ssi, err := s.sectorsForProof(ctx, good, toProve, ts)
```

Comment:
`partition.Sectors` included terminated sectors.. actor code expects `Sectors - Terminated - (Faults - Recoveries)`

Reply:
I'm not sure what was wrong here, but this isn't the right fix. We _should_ include terminated sectors in this bitfield. `sectorsForProof` will (or should) replace these terminated sectors (really, all sectors _not_ included in "good") with the first "good" sector.

See https://github.com/filecoin-project/specs-actors/blob/d48c45d15f1d79b06bc27c5be326a9b88c1fae39/actors/builtin/miner/deadline_state.go#L961